Sources/NIO/Utilities.swift

```diff
@@ -23,6 +23,18 @@ internal func debugOnly(_ body: () -> Void) {
     assert({ body(); return true }())
 }
 
+@inline(__always)
+private func isRelease() -> Bool {
+    return _isDebugAssertConfiguration()
```

Comment:
Actually isn't *this* backwards? The naming here is a bit confusing.

Reply:
Regardless, the condition we want is `debugOnly`, not `releaseOnly`, so I don't think we need this change.

Reply:
thanks @Lukasa , updated